isaric/parsers/isaric-ccpuk.toml

```diff
@@ -80,6 +80,11 @@
       { field = "ethnic___10", values = { 1 = "Unknown" } },
     ]
 
+  [subject.works_lab]
+    field = "labwork_erterm"
+    ref = "Y/N/NK"
+    description = "Employed in a microbiology laboratory?"
```

Comment:
same comment mapping Y responses only to works_lab from the microbiology question